src/pocketmine/event/block/SignChangeEvent.php

```diff
@@ -73,6 +73,13 @@ public function getLine($index){
 	}
 
 	/**
+	 * @param string[] $lines
+	 */
+	public function setLines(array $lines){
+		$this->lines = $lines;
```

Comment:
it wouldn't help much. You could still be supplying 1=>"lol" 402=>"aah" 94=>"l" 93=>"la"